libraries/core-react/src/components/Breadcrumbs/Breadcrumb.tsx

```diff
@@ -38,10 +38,20 @@ type BreadcrumbProps = {
   onClick?: (e: MouseEvent | KeyboardEvent) => void
   /** Children is breadcrumb text */
   children: string
-} & HTMLAttributes<HTMLDivElement>
+} & React.AnchorHTMLAttributes<HTMLAnchorElement>
 
-export const Breadcrumb = forwardRef<HTMLDivElement, BreadcrumbProps>(
+export const Breadcrumb = forwardRef<HTMLAnchorElement, BreadcrumbProps>(
   function Breadcrumb({ children, maxWidth, ...other }, ref) {
+    const [openState, setOpenState] = React.useState(false)
```

Comment:
Suggest renaming this property for what it controls, which is the `Tooltip`, so maybe `openTooltip` or something like that?